### PR TITLE
Issue 1451

### PR DESCRIPTION
**Pull request by antonkatz**

### CODE REVIEW

File: components/navigation/ui/src/main/resources/XWiki/SearchSuggestConfig.xml [hunk 1]
```diff
@@ -289,6 +289,12 @@
       </sectionPriority>
     </class>
     <property>
+      <advancedOnly/>
+    </property>
+    <property>
+      <categoryPriority/>
+    </property>
+    <property>
```
Comment (antonkatz) [inline components/navigation/ui/src/main/resources/XWiki/SearchSuggestConfig.xml]: Is this trash?

Comment (sdumitriu) [inline components/navigation/ui/src/main/resources/XWiki/SearchSuggestConfig.xml]: Not really, PhenoTips uses an extended version of the `ConfigurableClass`, with more fields than what XWiki has, while this document comes from XWiki. To keep the changes from the official document, we should not include these extra fields.

File: resources/ui/src/main/resources/PhenoTips/SuggestPatientsService.xml [hunk 2]
```diff
@@ -61,23 +61,84 @@
   #set ($query = $services.query.xwql($query))
   #set ($discard = $query.setLimit($nb))
   #set ($discard = $query.bindValue('input', "%${input}%"))
-  #set ($discard = $response.setContentType("text/xml"))
-  &lt;?xml version="1.0" encoding="UTF-8"?&gt;
   #set ($results = $query.execute())
-  &lt;results&gt;
+  #if ($request.permission)
+    #set ($requiredPermissions = "$request.permission")
```
Comment (sdumitriu) [inline resources/ui/src/main/resources/PhenoTips/SuggestPatientsService.xml]: There's no need to use a string here:

    #set ($requiredPermissions = $request.permission)

File: resources/ui/src/main/resources/PhenoTips/SuggestPatientsService.xml [hunk 3]
```diff
@@ -61,23 +61,84 @@
...
+  #else
+    #set ($requiredPermissions = "view")
+  #end
+  #if ($request.json)
+    $response.setContentType('application/json')
+    { "matchedPatients": [
+    #set ($first = true)
     #foreach ($item in $results)
-      #if ($xwiki.hasAccessLevel('view', $xcontext.user, $item))
+      #if ($xwiki.hasAccessLevel("$requiredPermissions", $xcontext.user, $item))
```
Comment (sdumitriu) [inline resources/ui/src/main/resources/PhenoTips/SuggestPatientsService.xml]: Don't string strings.

Comment (antonkatz) [inline resources/ui/src/main/resources/PhenoTips/SuggestPatientsService.xml]: But a stringed string is a string indeed

**Pull request merged**